styleguide/Components/Playground/PlaygroundRenderer.js

```diff
@@ -8,7 +8,7 @@ const PlaygroundRenderer = ({ name, preview, previewProps, tabBody, exampleIndex
     layout = true, // Обвесы в виде SplitLayout, SplitCol, etc
     iframe = true, // Рендерить пример в айфреме
     adaptivity: _adaptivity, // Продвинутые контролеры настроек адаптивности
-    webviewType = false, // Контролер настроек webviewType
+    showCustomPanelHeaderAfterProps = false, // Контроллер настроек связанных с пользовательским слотом `after` у `PanelHeader`
```

Comment:
```suggestion
    showCustomPanelHeaderAfterProps = false, // Контроллер настроек, связанных с пользовательским слотом `after` у `PanelHeader`
```
Таки потерялась запятая.)

Reply:
😭 😭 😭 

Сделяль ✅  4121a8fc68b21d86e1e0b4cab20d97863ebf542b )